examples/corim-unsigned-1.diag

```diff
@@ -0,0 +1,46 @@
+/ corim / 47110(
```

Comment:
I was planning to add the required header on a similar lines, using signed-corim-map ?

/ protected-signed-corim-header-map / {
/ corim.alg-id / 1 : 5,
/ corim.content-type / 3 : "application/rim+cbor",
/corim.issuer-key-id / 4 : b'0123456789',
/ corim.meta / 8 : { 
    / corim.signer / 0 : [
            {
            / corim.entity-name / x: "Acme Inc",
            / corim.reg-id / y: "http://acme.inc",
            /corim.role / z: 1 / corm.manifest-creator /
            },
            {
            / corim.entity-name / x: "Acme Inc",
            /corim.role / z: 2 / corm.manifest-signer /
            }
            ] 
        },
    },

Reply:
This example is in a file named `corim-unsigned-1.diag` :-) We will add a signed CoRIM in a separate `corim-signed-1.diag` example.

Reply:
Great!
I agree, I will mark the conversation as resolved!